doc/16-upgrading-icinga-2.md

```diff
@@ -8,6 +8,27 @@ Specific version upgrades are described below. Please note that version
 updates are incremental. An upgrade from v2.6 to v2.8 requires to
 follow the instructions for v2.7 too.
 
+## Upgrading to v2.13 <a id="upgrading-to-2-13"></a>
+
+### DB IDO Schema Update <a id="upgrading-to-2-13-db-ido"></a>
+
+There is an optional schema update on MySQL which increases the max length of object names from 128 to 255 characters.
+
+Please proceed here for the [MySQL upgrading docs](16-upgrading-icinga-2.md#upgrading-mysql-db).
+
+### Behavior changes <a id="upgrading-to-2-13-behavior-changes"></a>
+
+#### Deletion of child downtimes on services
+Service downtimes created while using the `all_services` flag on the [schedule-downtime](https://icinga.com/docs/icinga-2/snapshot/doc/12-icinga2-api/#schedule-downtime) API action will now automatically be deleted when deleting the hosts downtime. 
```

Comment:
... but not here?

Reply:
And here?

https://github.com/Icinga/icinga2/pull/8944/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR14-R15

Reply:
Fixed